test/system/src/test/java/io/pravega/test/system/AbstractReadWriteTest.java

```diff
@@ -552,6 +583,127 @@ private EventWriterConfig buildWriterConfig() {
         return validEvents;
     }
 
+    void writeInKVP(String scope, String kvtName, String kf, int kvpCount, KeyValueTableFactory keyValueTableFactory) {
+        log.info("Start KVP write operation of KVT {} and scope {}", kvtName, scope);
+        // Creating 5 threads to update KVP table entry
+        List<CompletableFuture<Void>> writerList = new ArrayList<>();
+        for (Integer i = 1; i <= kvpCount; i++) {
+            String value = convertStringToBinary("Hello World");
+            Integer keyId = i;
+            writerList.add(writeKVP(scope, kvtName, kf, value, keyId, keyValueTableFactory));
+        }
+        Futures.allOf(writerList);
```

Comment:
You need to wait on the result of this, otherwise its result is going to be lost. Use `join` or `get`

Reply:
If wait for result by using join then all the request run as sequential means Only writeInKVP() method running and readFromKVP() and performFailoverForTestsInvolvingKVS() waiting to finish writeInKVP().
But the test cases exception is writeInKVP() and readFromKVP() should run parallelly in background and scale up and scale down operation should happened through performFailoverForTestsInvolvingKVS() method

For writeInKVP() and readFromKVP() operation used data verification in every process

Reply:
So then what is the purpose of this line? It has absolutely no effect.

Furthermore, when do you know that your writes completed? How do you know they completed successfully or whether they ended up in an error?